modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,18 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.4.2 - 2023-04-26
+
+=== Fixed
+
+* Selection was not correctly scrolled horizontally into view when using the `selection.scrollIntoView` API.
+* Menu buttons would have the Tabstopping behaviour in toolbar.
+* The editor would display a notification error when it fails to retirieve a blob image uri.
+* Setting an invalid unit in the `fontsizeinput` would change it do the default value instead of reverting it back to the previous valid value.
+* The `urlinput` dialog component would not open the typeahead dropdown when the input value was reset to an empty string.
+* When dragging image elements and dropping the image in the editor the `dragend` event would sometimes not fire on firefox.
+* Context toolbars displayed the incorrect status for the advlist plugin buttons.
+* The image would not be inserted when using the `quickimage` button on Chrome.
```

Comment:
```suggestion
* Selection was not correctly scrolled horizontally into view when using the `selection.scrollIntoView` API.
* Tab navigation no longer incorrectly stops at menu buttons within toolbar groups
* The editor displayed a notification error when it failed to retrieve a blob image uri.
* Setting an invalid unit in the `fontsizeinput` changed it to the default value instead of reverting it to the previous, and valid, value.
* The `urlinput` dialog component would not open the type-ahead dropdown when the input value was reset to an empty string.
* When dragging image elements and dropping the image in the editor the `dragend` event would sometimes not fire on Firefox.
* Context toolbars displayed the incorrect status for the `advlist` plugin buttons.
* The image would not be inserted when the `quickimage` button was used on Chrome.
```

Multiple copy edits, and one re-write for meaning.